app/Routes.php

```diff
@@ -631,12 +634,12 @@ public static function getNamedRouteUrls( UrlGenerator $urlGenerator ): array {
 			'validateDonationAmount' => $urlGenerator->generateAbsoluteUrl( self::VALIDATE_DONATION_AMOUNT ),
 			'validateAddress' => $urlGenerator->generateAbsoluteUrl( self::VALIDATE_ADDRESS ),
 			'validateEmail' => $urlGenerator->generateAbsoluteUrl( self::VALIDATE_EMAIL ),
-			'validatePayment' => $urlGenerator->generateAbsoluteUrl( self::VALIDATE_DONATION_PAYMENT ),
+			'markEmptyValuesAsInvalid' => $urlGenerator->generateAbsoluteUrl( self::VALIDATE_DONATION_PAYMENT ),
```

Comment:
Oh no, this should really be `validatePayment`. Thankfully, it's deprecated and not used anywhere.